docs/tutorials/acme/quick-start/index.rst

```diff
@@ -354,12 +354,21 @@ install cert-manager. This example installed cert-manager into the
     # chart in the next step for `release-0.7` of cert-manager:
     $ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.7/deploy/manifests/00-crds.yaml
 
-    # Update your local Helm chart repositories
-    $ helm repo update
+    ## IMPORTANT: you MUST install the cert-manager CRDs **before** installing the
+    ## cert-manager Helm chart
+    $ kubectl apply \
+       -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.7/deploy/manifests/00-crds.yaml
 
-    # Install cert-manager
-    $ helm install --name cert-manager --namespace cert-manager stable/cert-manager
+    ## IMPORTANT: if the cert-manager namespace **already exists**, you MUST ensure
+    ## it has an additional label on it in order for the deployment to succeed
+    $ kubectl label namespace cert-manager certmanager.k8s.io/disable-validation="true"
 
+    ## Add the Jetstack Helm repository
+    $ helm repo add jetstack https://charts.jetstack.io
```

Comment:
I think we need a `helm repo update` here?

Reply:
:+1: yep i guess just incase they had the repo already added before?